project_setup/README.md

```diff
@@ -4,6 +4,27 @@ While the specific setup for each 18F project varies widely, there are certain
 elements that should be present on every project. This document aims to detail
 those elements and provide suggested tools and resources.
 
+### Initialization Checklist
+
+This list is aspirational, but a good place to start. We will try to include
+all in upcoming project templates; until then, do your best!
+
+1. Common project files (consider using [`18f init`](https://github.com/18F/18f-cli#18f-init)):
+    1. LICENSE.md
+    1. CONTRIBUTING.md
+    1. README.md
+    1. .about.yml
```

Comment:
An .nvmrc file would be great. I think we've standardized on NVM as our node version manager, and the nvmrc file would ensure everybody on a project is working on the same version  as node.